src/components/AsideHeader/components/CollapseButton/CollapseButton.module.scss

```
@@ -1,11 +1,49 @@
 @use '../../../variables';
+@use '../../../../../styles/mixins';
 
 $block: '.#{variables.$ns}collapse-button';
 
+$buttonHeight: 20px;
```

Stylelint (17.12.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)